inputfiles/removedTypes.jsonc

```diff
@@ -862,13 +862,6 @@
                     }
                 }
             },
-            "VideoEncoderConfig": {
-                "members": {
-                    "member": {
-                        "hevc": null // Blink only as of 2023-03
```

Comment:
This is still true, which makes this PR invalid.

Reply:
Closing this for now as above is true. Leaving here note for future reference that hevc codecs are supported on safari browser too, but for some reason they haven't included video encoder config parameters into the webkit browser engine webcodecs implementation.